### DISCUSSION TITLE
UDP Effect missing

### DISCUSSION THREAD

**Discussion by gsus24** [hyperion-project/hyperion.ng#1175] [Q&A] [answered]
Hello

installed today the new version but cannot find udp effect. I read its removed, because its not multi instance capable.
How to send an rgb byte[] with the count of leds to hyperion? To control every single led! JSON, Proto, Boblight?
JSON is for remote-control, Proto/Boblight for Imagedata. I have no idea.
Hope somebody have one.

Cheers

**Comment by Lord-Grey** [comment, marked as answer]
@gsus24 Not sure, if I really got what you would like to achieve. 
In case you would like to provide 1-n LED values (for one output instance), you can use JSON, PROTO and Flatbuffer.

Find here the documentation for JSON:  https://docs.hyperion-project.org/en/json/Control.html#set-color
Alternatively, you can use an Image as input which will be mapped to output LEDs, as per Hyperion standard capabilities....

**Reply by gsus24** [reply]
@Lord-Grey thanks for fast reply. I used this project on android with the old version of hyperion and upd effect
[https://github.com/doktormerlin/HyperionVisualizer/blob/4ef594fd259f577972aaa805a6e6662e69cb0bfc/app/src/main/java/com/merlbot/hyperionvisualizer/VisualizerService.java#L260](url)

Simplest way i think will be send an color array use JSON.
Are you fimilar with, is it a JSON-RPC Server or REST API with JSON package?  Found this library for android. 
[https://github.com/resourcepool/jarpic-client](url)

**Reply by Lord-Grey** [reply]
see https://docs.hyperion-project.org/en/json/#what-is-json
